tests/e2e/specs/modules/optimize/activation.test.js

```diff
@@ -79,20 +79,7 @@ describe( 'Optimize Activation', () => {
 	beforeAll( async () => {
 		await page.setRequestInterception( true );
 		useRequestInterception( ( request ) => {
-			if ( request.url().match( '/google-site-kit/v1/data/' ) ) {
-				request.respond( { status: 200 } );
-			} else if (
-				request
-					.url()
-					.match(
-						'/wp-json/google-site-kit/v1/modules/analytics/data/report?'
-					)
-			) {
-				request.respond( {
-					status: 200,
-					body: JSON.stringify( { placeholder_response: true } ),
-				} );
-			} else if (
```

Comment:
Looks like this has been deleted by accident. Let's revert this.

Reply:
It was in the commit linked in the IB https://github.com/google/site-kit-wp/commit/d77d4fc8d20ff6d5fa5e266c9be495c3bc3d8ae3

> * [Remove all E2E request inceptions for unused `v1/data` API calls](https://github.com/google/site-kit-wp/commit/d77d4fc8d20ff6d5fa5e266c9be495c3bc3d8ae3)

Reply:
Re-added it in https://github.com/google/site-kit-wp/pull/3823/commits/9c273d6fe57af3e5b1779779bc64eb19b81eab08

Reply:
@ivankruchkoff you also added request interception for `v1/data` endpoint. We don't need it, so it should be re-removed.